irohad/consensus/yac/transport/yac_pb_converters.hpp

```diff
@@ -116,9 +116,13 @@ namespace iroha {
           auto deserialize = [&](auto &pubkey,
                                  auto &signature,
                                  const auto &msg) {
+            auto const &pubkey_hex = bytestringToHexstring(pubkey);
+            auto const &signature_hex = bytestringToHexstring(signature);
```

Comment:
doesn't it bind to a temporary?
here and in other places

Reply:
https://herbsutter.com/2008/01/01/gotw-88-a-candidate-for-the-most-important-const/
But looks like it can be simplified when modern compilers are used.